dropwizard-app/src/main/java/org/coner/boundary/HandicapGroupBoundary.java

```diff
@@ -0,0 +1,57 @@
+package org.coner.boundary;
+
+import org.coner.core.domain.HandicapGroup;
+
+/**
+ *
```

Comment:
Blank javadoc again? I'll research if we can implement a rule to catch this. I think I've probably made a few of these myself